Configurations/15-ios.conf

```
@@ -34,6 +34,29 @@ my %targets = (
         inherit_from     => [ "ios-common" ],
         CC               => "xcrun -sdk iphonesimulator cc",
     },
+    "iossimulator-arm64-xcrun" => {
+        inherit_from     => [ "ios-common" ],
+        CC               => "xcrun -sdk iphonesimulator cc",
+        cflags           => add("-arch arm64 -mios-simulator-version-min=7.0.0"),
```

We should emulate what actual iOS uses
```suggestion
        cflags           => add("-arch arm64 -mios-simulator-version-min=7.0.0 -fno-common"),
```

---

Good point.